Source/render/NodeRenderer.swift

```diff
@@ -21,6 +21,29 @@ class NodeRenderer {
     fileprivate var active = false
     weak var animationCache: AnimationCache?
 
+    fileprivate var cachedAbsPlace: Transform?
+    public var absPlace: Transform {
+        get {
+            if let place = cachedAbsPlace {
+                return place
+            }
+
+            if let place = parentRenderer?.absPlace.concat(with: node.place) {
+                cachedAbsPlace = place
+                return place
+            }
+
+            return .identity
+        }
+    }
+    func freeCachedAbsPlace() {
+        cachedAbsPlace = nil
+    }
+
+    open var node: Node {
```

Comment:
Why is it `var`? I think it can be replaced with `let` here and in all subclasses.

Reply:
That's cause current animations use replaceNode method of renderers - not to change actual initial node during animation. We should probably think of something different, but for now I can't make it immutable

Reply:
Got it. Should be removed in future, but okay for now.